src/ui/layouts.tsx

```diff
@@ -70,16 +69,10 @@ export const OrgLayout = ({
           mb: '$xl',
         }}
       >
-        <ApeAvatar
-          alt="organization"
-          src="/imgs/avatar/placeholder.jpg"
-          style={{
-            width: 36,
-            height: 36,
-            borderRadius: '50%',
-            border: '1px solid rgba(94, 111, 116, 0.7)',
-            marginRight: '16px',
-          }}
+        <Avatar
+          name="organization"
```

Comment:
here, `name` should be the organization's name

Reply:
I can't reach the Organization's Name

Reply:
hmm, this layout should actually be removed entirely. will see if i can fix this in a different PR